build/targets/nuget-common.props

```diff
@@ -1,7 +1,6 @@
 <Project>
     <PropertyGroup>
         <Authors>The OpenTK team.</Authors>
```

Comment:
I originally left it in there since VS's description of this element said it was for NuGet:
![](https://i.imgur.com/wZxEkIp.png)

Could be moved anyways, though.

Reply:
Alright, let's keep it there then.